File: DebtAngular/Models/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DebtAngular.Models
{
    public class Task : CommonModel
    {
        public string Name { get; set; }
        public double Sum { get; set; }

        public string UserId { get; set; }

        public virtual ICollection<Member> Members { get; set; }
        public virtual ICollection<Debt> Debts { get; set; }
    }
}

File: DebtAngular/ClientApp/src/app/_models/task.ts
import { Debt } from './debt';
import { Member } from "./member";

export class Task {
    id: string;
    name: string;
    sum: number;
    userId: string;
    members: Member[] = [];
    debts: Debt[] = [];
    constructor() {
    this.id = '00000000-0000-0000-0000-000000000000';
  }

}
